### ISSUE TITLE
[feature request] Stream quality selection

### ISSUE THREAD

**Issue by coxackie** [xou816/spot#130]
Right now, one cannot select what stream quality to play the songs. Could this be please be implemented? (along with potentially other options?) I know that spotifyd allows this, so I am pretty sure it is doable with librespot intergration.

 Great work by the way. This is awesome and (almost too) lightweight.

**Comment by xou816** [comment]
Hi! Thanks for the kind words :) This is not implemented in librespot, couldn't find any mention of it in their issue tracker either. No idea if this is feasible or not, but I can't help with that -- not knowledgeable about the inner workings of librespot!

**Comment by coxackie** [comment]
hmmm... @xou816  from a quick google search:
https://github.com/librespot-org/librespot/wiki/Options
look at the line
```
Option | b | bitrate | Bitrate (96, 160 or 320). Defaults to 160 | BITRATE
```
So, presumably the sampling rate right now is sub-optimal... could you please have a better look, if possible?

**Comment by xou816** [comment]
Ah, I misunderstood! Sure, I (or someone else) might have a look later :)

**Comment by xou816** [comment]
Fixed in 0.1.10, can be changed with gsettings. Closing this for now, feel free to reopen if needed. I'll add an issue to implement a GUI for these settings.
